### DISCUSSION TITLE
Disculpen ,para la kata 2 solo agregamos captura de pantalla del ejercio realizado .

### DISCUSSION THREAD

**Discussion by JoseBenin82** [LaunchX-InnovaccionVirtual/CursoIntroPython#195] [Q&A]
:)

**Comment by JossPerezT** [comment]
Si, así es. Puras capturas de la consola.

**Reply by JoseBenin82** [reply]
SUPER, ¡GRACIAS!

**Comment by DanielOe21** [comment]
# Kata 2
Hola @JoseBenin82 ! La kata es replicar los ejemplos que vienen ahí

Puedes crear un archivo de texto Markdown con capturas de pantalla del proceso que haz realizado, o una libreta de Jupyter (Que tambien utiliza Markdown).

Espero haberte ayudado, por favor no olvides marcar tu pregunta como CONTESTADA si te parece que ha sido así.

### Follow me!
<a href="https://github.com/DanielOe-21">
<img alt="GitHub followers" src="https://img.shields.io/github/followers/DanielOe-21?style=social">
</a>

**Reply by JoseBenin82** [reply]
gracias amigo

**Comment by PlayerOneV** [comment]
Como evidencia yo solamente cree el entorno virtual en el directorio del modulo 2.

**Comment by Ingrzs** [comment]
es literal subir una imagen de captura o subir una Markdown y ahi dentro la captura? Ya me confundieron

**Reply by Nukario** [reply]
cualquiera de las dos, como lo prefieras puedes poner solo la captura, o darle formato dentro de un markdown en jupiter, cualquiera de las dos formas cuentan

**Reply by JSantiagomx** [reply]
como insertas imagenes en un Markdown usando visual studio code?

**Reply by JorgeLReyes** [reply]
> como insertas imagenes en un Markdown usando visual studio code?

A mi me funcionó de esta manera < img src="File/imagen.png"> 
pd: el espacio entre < img quítalo lo puse así porque al comentar me ponia directamente una imagen rota, espero te sirva

**Reply by JSantiagomx** [reply]
lo puse asi:
< img src="File/venv.jpg">
en un markdown en blanco, pero cuando le doy run all me aparece el icono de una imagen cortada, porque sera. Tengo la imagen en la carpeta en donde estan mis ejercicios.
Gracias.

**Reply by JorgeLReyes** [reply]
> lo puse asi: < img src="File/venv.jpg"> en un markdown en blanco, pero cuando le doy run all me aparece el icono de una imagen cortada, porque sera. Tengo la imagen en la carpeta en donde estan mis ejercicios. Gracias.

En mi caso las agregue a una subcarpeta donde guardo todas las katas
![image](https://user-images.githubusercontent.com/99159473/153805983-447eff92-dce3-4ada-a458-e8110b6065fd.png)

Y en markdown simplemente puse esto
![image](https://user-images.githubusercontent.com/99159473/153806097-b7e97c8c-c64d-4c3a-96b6-187b6fefa072.png)
< img src="NombreCarpeta/Imagen.pgn">
<img va junto pero si lo pongo asi aqui solo veras una imagen cortada

**Reply by KarenPamelaLopezMedina** [reply]
Hola, a mí me funció esta parte de código: 

import IPython.display as display
from PIL import Image
display.display(Image.open('C:/Users/super/Desktop/Katas/02-Kata/capt1.png'))

**Reply by KarenPamelaLopezMedina** [reply]
Este es el link de donde obtuve la información https://www.delftstack.com/es/howto/python/python-display-image/#utilice-el-m%25C3%25B3dulo-pil-para-mostrar-una-imagen-en-python
